.github/workflows/smoketest.yml

```diff
@@ -38,11 +38,11 @@ jobs:
             ${{ runner.os }}-yarn-
 
       - name: Install dependencies on cache miss
-        run: yarn install --frozen-lockfile
+        run: yarn install
```

Comment:
Immutable is the replacement for frozen lockfile.

```suggestion
        run: yarn install --immutable
```

Reply:
Yeah I did that. The build had a checksum error I couldn't solve.

Reply:
See 25b28fbea5e519f1e7256da80805310a11ccfd5c

Reply:
Likely needs a lockfile generated on Linux.

Reply:
VSCodespaces to the rescue then!